javascript/javascript1/week1/dogage.js

```diff
@@ -0,0 +1,10 @@
+let a = prompt("dogYearOfBirth");
+let b = prompt("dogYearFuture");
+shouldShowResultInDogYears = true;
+if (shouldShowResultInDogYears) {
```

Comment:
There is no variable called `shouldShowResultInDogYears` I think you forget to declare it